java/code/src/com/suse/manager/model/attestation/ServerCoCoAttestationConfig.java

```diff
@@ -32,7 +34,7 @@
 
 @Entity
 @Table(name = "suseServerCoCoAttestationConfig")
-public class ServerCoCoAttestationConfig {
+public class ServerCoCoAttestationConfig implements Serializable  {
```

Comment:
Here and in all classes implementing `Serializable`, according to the spec it's _strongly recommended_ to also add a `serialVersionUUID` field. The problem is always maintaining it as it should be changed every time a serialization-breaking change is implemented :sweat_smile:

Reply:
I enabled the checking of it in intellij, and it's missing almost everywhere ... I think it would be better to address it in another PR